internal/commands/update_buildpack.go

```diff
@@ -60,33 +60,20 @@ func UpdateBuildpackCommand(ctx context.Context) (err error) {
 
 	logger.Debug().Msgf("identified remote: %s, %s", codecraftersRemote.Name, codecraftersRemote.Url)
 
-	logger.Debug().Msg("reading and updating codecrafters.yml file")
-
-	codecraftersYmlPath := filepath.Join(repoDir, "codecrafters.yml")
+	logger.Debug().Msg("fetching current buildpack from server")
 
-	content, err := os.ReadFile(codecraftersYmlPath)
-	if err != nil {
-		if os.IsNotExist(err) {
-			return fmt.Errorf("codecrafters.yml file not found in repository root")
-		}
-		return fmt.Errorf("failed to read codecrafters.yml: %w", err)
-	}
-
-	updatedContent := utils.ReplaceYAMLField(string(content), "language_pack", "buildpack")
+	codecraftersClient := utils.NewCodecraftersClient(codecraftersRemote.CodecraftersServerURL())
 
-	err = os.WriteFile(codecraftersYmlPath, []byte(updatedContent), 0644)
+	repositoryBuildpackResponse, err := codecraftersClient.FetchRepositoryBuildpack(codecraftersRemote.CodecraftersRepositoryId())
 	if err != nil {
-		return fmt.Errorf("failed to write updated codecrafters.yml: %w", err)
+		logger.Debug().Err(err).Msg("failed to fetch repository buildpack")
+		return fmt.Errorf("failed to fetch repository buildpack: %w", err)
 	}
 
-	buildpackValue := utils.ExtractYAMLFieldValue(updatedContent, "buildpack")
-	if buildpackValue == "" {
-		return fmt.Errorf("buildpack value not found in codecrafters.yml")
-	}
+	buildpackValue := repositoryBuildpackResponse.Buildpack.Slug
```

Comment:
```suggestion
	currentBuildpackSlug := repositoryBuildpackResponse.Buildpack.Slug
```

(We already have the term "slug" for this, "value" isn't any more descriptive)